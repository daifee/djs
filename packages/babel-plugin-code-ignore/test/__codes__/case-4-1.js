import React from 'react';

// 问答框
function Dialog() {

  return (<div className="dialog" />)
}

function Main() {

  return (
    <div id="main">
      {/* @code-ignore-next-jsx-element kuwo */}
      <Dialog id="global" key="global" />
      <div>1111111</div>
      <Dialog>222222</Dialog>
      {/* @code-ignore-next-jsx-element kuwo */}
      <>
        <div>333333333</div>
      </>
    </div>
  );
}