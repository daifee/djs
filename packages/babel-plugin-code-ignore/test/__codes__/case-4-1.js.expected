"use strict";

var _react = _interopRequireDefault(require("react"));
function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }
// 问答框
function Dialog() {
  return /*#__PURE__*/_react.default.createElement("div", {
    className: "dialog"
  });
}
function Main() {
  return /*#__PURE__*/_react.default.createElement("div", {
    id: "main"
  }, /*#__PURE__*/_react.default.createElement("div", null, "1111111"), /*#__PURE__*/_react.default.createElement(Dialog, null, "222222"), /*#__PURE__*/_react.default.createElement(_react.default.Fragment, null, /*#__PURE__*/_react.default.createElement("div", null, "333333333")));
}